src/generic/video-modal/VideoModal.scss

```diff
@@ -0,0 +1,4 @@
+.pgn__modal {
```

Comment:
[question] Should we apply this style for all `.pgn__modal` or we need to wrap it in a specific class name for localize influence?